src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -502,6 +517,7 @@ protected boolean prepare(final O operation) {
 	@Override
 	protected final boolean submit(final O op)
 					throws IllegalStateException {
+		boolean eventOpHasFailed = false;
```

Comment:
boolean ok = concurrencyThrottle.tryAcquire();

Reply:
I do not have a better suggestion for now, but isn't "ok" a bit too vague?